ansible/deploy-proteus-notify.yml

```diff
@@ -0,0 +1,31 @@
+- hosts: notify.proteus.test.ooni.io
+  vars:
+    proteus_notify_https_port: 443
+    letsencrypt_nginx: yes
+    letsencrypt_domains: "{{ inventory_hostname }}"
+    proteus_database_url: "postgres://proteus:{{ proteus_database_password_testing }}@db-1.proteus.test.ooni.io:5432/proteus?sslmode=require"
+    proteus_auth_jwt_secret : "{{ proteus_auth_jwt_secret_testing }}"
+    proteus_fcm_server_key: "{{ fcm_server_key_testing }}"
+    proteus_apn_key_password: "{{ proteus_apn_key_password_testing }}"
+    proteus_apn_key_content: "{{ proteus_apn_key_content_testing }}"
+    proteus_environment: "production"
+    proteus_notify_basic_auth_password: "{{ proteus_notify_basic_auth_password_testing }}"
```

Comment:
> Though while deploying a first version of production I accidentally used the testing password in production and to avoid the risk of confusion I found it more clear to just label them as such.

Yeah, I guess this is a good argument in favour of a little more redundancy and robustness.